test/system/015-help.bats

```diff
@@ -136,7 +136,7 @@ function check_help() {
             run_podman '?' "$@" $cmd </dev/null
             is "$status" 125 \
                "'$usage' indicates at least one required arg. I invoked it with no args and expected an error exit code"
-            is "$output" "Error:.* \(require\|specif\|must\|provide\|need\|choose\|accepts\)" \
+            is "$output" "Error:.* \(require\|specify\|must\|provide\|need\|choose\|accepts\)" \
```

Comment:
This one needs to revert: error messages can be "specify", "specified", or other possibilities.

Reply:
/me thinks you forgot to revert this. Tests are gonna fail again.